### ISSUE TITLE
Less sources are stuck with Bower

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#5]
While trying to compile sources directly with Less just after downloading the zip file, I couldn't because of the way Bootstrap files are called in [bootstrap-orange2015.less](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/master/app/less/bootstrap-orange2015.less). 

This file imports Bootstrap sources from `bower_components` folder, which doesn't exist in the case we don't install Boosted from Bower. I saw the same issue (if this is one) in the gruntfile.

I see you're working on a Sass version of Boosted, and files aren't imported from `bower_components`.

Sadly I downloaded sources three weeks ago and wanted to only use the Less source files to avoid overriding some configurable Bootstrap things. It wasn't possible at that time without spending time to manually download Bootstrap and modify imports.

I think it could be great to fix this while working on the Sass version. Or is this acceptable? This is not a real issue since installing from bower does work, but this doesn't allow any other way of installing Boosted.

**Comment by ffoodd** [comment]
I just cloned the repo, and I'm able to mitigate this issue: in the doc, mentioning it's needed to run `bower install` after downloading the source code (instead of saying bower is an alternative) should prevent noobs like me to get the error.

Since you're already working on the next release, this could do the job.

Alternative (and technical) thought: there might be a way to hook a script when running `npm install`? Since `npm install` is required anyway, make it handle Bower too should be simpler. We could have a look to [the npm documentation](https://docs.npmjs.com/misc/scripts). Writing node scripts is pretty easy ([as I did](https://github.com/ffoodd/a11y.css/blob/68871ffc1e6502a2906fed00c955c9bf7b5404ec/package.json#L11)) and if we manage to hook checking for a bower version/install bower globally then running `bower install` automagically, that could work.
